pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -313,7 +313,7 @@ func (n *NetworkPolicyController) createAppliedToGroup(np *networkingv1.NetworkP
 		UID:      types.UID(appliedToGroupUID),
 		Selector: *groupSelector,
 	}
-	klog.V(2).Infof("Creating new AppliedToGroup %s", newAppliedToGroup.Name)
+	klog.V(2).Infof("Creating new AppliedToGroup %s with selector (%s)", newAppliedToGroup.Name, newAppliedToGroup.Selector.NormalizedName)
```

Comment:
do you also want to remove the log at L307?

Reply:
Done